tests/integration/utils.py

```diff
@@ -93,6 +94,34 @@ def wait_for_complete_request(self, response):
 
         return response
 
+    def fetch_all_requests(self, request_json={}):
+        """
+        Fetch a list of requests from the Cachito API.
+
+        :return: Object that contains response from the Cachito API
+        :rtype: list
+        """
+        url_values = urllib.parse.urlencode(request_json)
+        result_url = f"{self._cachito_api_url}/requests?{url_values}"
+        resp = requests.get(result_url)
+        resp.raise_for_status()
+
+        page = request_json.get("page")
+        per_page = request_json.get("per_page")
+        if all(parameter is None for parameter in [page, per_page]):
```

Comment:
Optional: It's simpler to read the following in my opinion since you're only comparing to values.
```suggestion
        if page is None and per_page is None:
```